routes/post.js

```diff
@@ -35,7 +29,7 @@ module.exports = function(app) {
     app.get('/posts/all', function(req, res, next) {
 
         // might want to limit posts to certain time period
-        Post.find().exec(function(err, posts) {
+        postServices.findPost = function(err, posts) {
```

Comment:
this shouldn't be a function declaration, you're calling it w/ the callback

do `postServices.findPost(function(err, posts ...`